Core/include/Acts/Surfaces/RadialBounds.hpp

```diff
@@ -31,7 +31,8 @@ class RadialBounds : public DiscBounds {
     eMaxR = 1,
     eHalfPhiSector = 2,
     eAveragePhi = 3,
-    eSize = 4
+    eAngle = 4,
```

Comment:
Why do the `RadialBounds` change ?

For the new CylinderShape we might need instead something like `RadialEllipsoidBounds` or can we get them into one hood (actually I think we could live with `EllipsoidBounds` and keep the coordinate system cartesian).

Reply:
This was a mistake, left over by a trial implementation of bevelling of the "side"-disc but in the end I didn't need it but I forgot to remove it